lib/macho/load_commands.rb

```diff
@@ -204,15 +97,30 @@ def to_s
     # pretend that strings stored in LCs are immediately available without
     # explicit operations on the raw Mach-O data.
     class LCStr
+      # @return [Fixnum] the number of null bytes used to pad the string
+      attr_reader :npad
```

Comment:
Agreed. My only justification was that `npad` only *needs* to be calculated once per `LCStr` instance, and that's hardly a significant performance concern.